src/GridList/GridTileTitlebar.js

```diff
@@ -0,0 +1,174 @@
+// @flow weak
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from '../utils/customPropTypes';
+
+export const styleSheet = createStyleSheet('MuiGridTileTitlebar', (theme) => {
+  const { mixins } = theme;
```

Comment:
I have been removing all those partners ` const { x, y, z } = theme` as I think that it's making static analysis of how the theme is used much easier (critical IMHO to be able to refactor and make changes with confidence).
What do you think about it?